Add DappTools and Foundry starter kits (#656)
smartcontractkit/documentation: docs/Developer Reference/create-a-chainlinked-project.md

cd hardhat-starter-kit
```

For instructions on how to use the starter kit, refer to the [Hardhat starter kit README](https://github.com/smartcontractkit/hardhat-starter-kit/blob/main/README.md).

> For more details on how to use Chainlink with Hardhat, see our blog post [How to use Hardhat with Chainlink](https://blog.chain.link/using-chainlink-with-hardhat/).

Reviewer: ```suggestion
For more details on how to use Chainlink with Hardhat, see our blog post about [How to use Hardhat with Chainlink](https://blog.chain.link/using-chainlink-with-hardhat/).
```

brownie bake chainlink-mix
cd chainlink-mix
```
For instructions on how to use the starter kit, refer to the [Brownie starter kit README](https://github.com/smartcontractkit/chainlink-mix/blob/master/README.md).

> For more details on how to use Chainlink with Brownie, see our blog post [Develop a DeFi Project Using Python](https://blog.chain.link/develop-python-defi-project/).

Reviewer: ```suggestion
For more details on how to use Chainlink with Brownie, see the [Develop a DeFi Project Using Python](https://blog.chain.link/develop-python-defi-project/) blog post.
```

truffle unbox smartcontractkit/truffle-starter-kit
```

For instructions on how to use the starter kit, refer to the [Truffle starter kit README](https://github.com/smartcontractkit/truffle-starter-kit/blob/master/README.md).

For more details on how to use Chainlink with Truffle, see our blog post [Using Truffle to interact with Chainlink Smart Contracts](https://www.trufflesuite.com/blog/using-truffle-to-interact-with-chainlink-smart-contracts).
___

> ## DappTools Starter Kit
> 
> To learn more about DappTools, refer to the [DappTools Documentation](https://dapp.tools/). You can also refer to the [Installation instructions](https://github.com/dapphub/dapptools#installation).
> 
> Once installed, clone the starter kit and install the project's dependencies:
> 
> ```shell
> git clone https://github.com/smartcontractkit/dapptools-starter-kit
> cd dapptools-starter-kit
> make # This installs the project's dependencies.
> ```
> 
> For instructions on how to use the starter kit, refer to the DappTools starter kit [README](https://github.com/smartcontractkit/dapptools-starter-kit#readme).
> 
> For more details on how to use Chainlink with DappTools, see our blog post [How To Use DappTools](https://blog.chain.link/how-to-use-dapptools/).

Reviewer: ```suggestion
## DappTools Starter Kit

To learn more about DappTools, refer to the [DappTools Documentation](https://dapp.tools/).

1. Install Dapp tools using the [Installation instructions](https://github.com/dapphub/dapptools#installation) in the Dapp tools GitHub repository.

1. After you install the tools, clone the starter kit and install the dependencies:

      ```shell
      git clone https://github.com/smartcontractkit/dapptools-starter-kit
      cd dapptools-starter-kit
      make # This installs the project's dependencies.
      ```

For instructions on how to use the starter kit, refer to the DappTools starter kit [README](https://github.com/smartcontractkit/dapptools-starter-kit#readme).

For more details on how to use Chainlink with DappTools, see our blog post [How To Use DappTools](https://blog.chain.link/how-to-use-dapptools/).
```

___

## Foundry Starter Kit

> To learn more about Foundry, refer to the [Foundry Documentation](https://onbjerg.github.io/foundry-book/). You you also refer to the [Installation instructions](https://onbjerg.github.io/foundry-book/getting-started/installation.html).
> 
> Once installed, clone the starter kit and install the project's dependencies:
> 
> ```shell
> git clone https://github.com/smartcontractkit/foundry-starter-kit
> cd foundry-starter-kit
> make # This installs the project's dependencies.
> ```
> 
> For instructions on how to use the starter kit, refer to the [Foundry starter kit README](https://github.com/smartcontractkit/foundry-starter-kit#readme).

Reviewer: ```suggestion
To learn more about Foundry, refer to the [Foundry Documentation](https://onbjerg.github.io/foundry-book/).

1. Install Foundry using the [Installation instructions](https://onbjerg.github.io/foundry-book/getting-started/installation.html) on GitHub.io.

1. After you install Foundry, clone the starter kit and install the projects dependencies:

      ```shell
      git clone https://github.com/smartcontractkit/foundry-starter-kit
      cd foundry-starter-kit
      make # This installs the project's dependencies.
      ```

For instructions on how to use the starter kit, refer to the [Foundry starter kit README](https://github.com/smartcontractkit/foundry-starter-kit#readme).
```

truffle unbox smartcontractkit/truffle-starter-kit
```

For instructions on how to use the starter kit, refer to the [Truffle starter kit README](https://github.com/smartcontractkit/truffle-starter-kit/blob/master/README.md).

> For more details on how to use Chainlink with Truffle, see our blog post [Using Truffle to interact with Chainlink Smart Contracts](https://www.trufflesuite.com/blog/using-truffle-to-interact-with-chainlink-smart-contracts).

Reviewer: ```suggestion
For more details on how to use Chainlink with Truffle, see our blog post about [Using Truffle to interact with Chainlink Smart Contracts](https://www.trufflesuite.com/blog/using-truffle-to-interact-with-chainlink-smart-contracts).
```

___

## DappTools Starter Kit

To learn more about DappTools, refer to the [DappTools Documentation](https://dapp.tools/).

1. Install Dapp tools using the [Installation instructions](https://github.com/dapphub/dapptools#installation) in the Dapp tools GitHub repository.

1. After you install the tools, clone the starter kit and install the dependencies:

      ```shell
      git clone https://github.com/smartcontractkit/dapptools-starter-kit
      cd dapptools-starter-kit
      make # This installs the project's dependencies.
      ```

For instructions on how to use the starter kit, refer to the DappTools starter kit [README](https://github.com/smartcontractkit/dapptools-starter-kit#readme).

> For more details on how to use Chainlink with DappTools, see our blog post [How To Use DappTools](https://blog.chain.link/how-to-use-dapptools/).

Reviewer: ```suggestion
For more details on how to use Chainlink with DappTools, see the [How To Use DappTools](https://blog.chain.link/how-to-use-dapptools/) blog post.
```

___

## Foundry Starter Kit

To learn more about Foundry, refer to the [Foundry Documentation](https://onbjerg.github.io/foundry-book/).

1. Install Foundry using the [Installation instructions](https://onbjerg.github.io/foundry-book/getting-started/installation.html) on GitHub.io.

> 1. After you install Foundry, clone the starter kit and install the projects dependencies:

Reviewer: ```suggestion
1. After you install Foundry, clone the starter kit and install the project dependencies:
```
